libs/auth-api-lib/src/lib/delegations/models/delegation.model.ts

```diff
@@ -21,6 +21,7 @@ import { DelegationScope } from './delegation-scope.model'
 import { Domain } from '../../resources/models/domain.model'
 import { MergedDelegationDTO } from '../dto/merged-delegation.dto'
 import { DelegationType } from '../types/delegationType'
+import max from 'lodash/max'
```

Comment:
~ External package dependencies should normally be above libs and relative imports. We need to look into using ESlint to force import order.